CMakeLists.txt

```diff
@@ -235,7 +230,7 @@ if(${CMAKE_VERSION} VERSION_LESS ${FUTURE_MINIMUM_CMAKE_VERSION})
     "is deprecated and will be removed.")
 endif()
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
This also needs to be set in FindOpenVDB.cmake with the `INTERFACE_COMPILE_FEATURES` property (right at the bottom of the file during target creation)

Reply:
Good catch, thanks